holidays/countries/argentina.py

```diff
@@ -24,94 +24,260 @@
+        ),
+        2014: (
+            (MAY, 2, arg_bridge_public_holiday),
+            (DEC, 26, arg_bridge_public_holiday),
+        ),
+        2015: (
+            (MAR, 23, arg_bridge_public_holiday),
+            (DEC, 7, arg_bridge_public_holiday),
+        ),
+        2016: (
+            (JUL, 8, arg_bridge_public_holiday),
+            (DEC, 9, arg_bridge_public_holiday),
+        ),
+        2018: (
+            (APR, 30, arg_bridge_public_holiday),
+            (DEC, 24, arg_bridge_public_holiday),
+            (DEC, 31, arg_bridge_public_holiday),
+        ),
+        2019: (
+            (JUL, 8, arg_bridge_public_holiday),
+            (AUG, 19, arg_bridge_public_holiday),
+            (OCT, 14, arg_bridge_public_holiday),
+        ),
+        2020: (
+            (MAR, 23, arg_bridge_public_holiday),
+            (JUL, 10, arg_bridge_public_holiday),
+            (DEC, 7, arg_bridge_public_holiday),
+        ),
+        2021: (
+            (MAY, 24, arg_bridge_public_holiday),
+            (OCT, 8, arg_bridge_public_holiday),
+            (NOV, 22, arg_bridge_public_holiday),
+        ),
+        2022: (
+            (MAY, 18, national_census_2022),
+            (OCT, 7, arg_bridge_public_holiday),
+            (NOV, 21, arg_bridge_public_holiday),
+            (DEC, 9, arg_bridge_public_holiday),
+        ),
+        2023: (
+            (MAY, 26, arg_bridge_public_holiday),
+            (JUN, 19, arg_bridge_public_holiday),
+            (OCT, 13, arg_bridge_public_holiday),
         ),
     }
-    default_language = "es"
 
     def _populate(self, year):
-        # The "movable holidays" whose dates coincide with Tuesdays and
-        # Wednesdays will be moved to the previous Monday. Those that
-        # coincide with Thursday, Friday, Saturday and Sunday will be
-        # moved to the following Monday.
+        """
+        !!! Movable Holidays Law !!!
+
+        - Decreto 1584/2010: 2010-11-03
+            - AUG 17, OCT 12, NOV 20 Holidays will always be on MON
+        - Decreto 52/2017: 2017-01-23 (Reconfirmed in Ley 27399)
+            - If TUE/WED ← observed on previous MON
+            - If THUR/FRI ← observed on next MON
+        """
+
         def _add_movable(hol_date: date, hol_name: str) -> None:
             if self.observed:
                 if self._is_tuesday(hol_date) or self._is_wednesday(hol_date):
                     hol_date += rd(weekday=MO(-1))
                     hol_name = self.tr("%s (Observado)") % hol_name
-                elif not self._is_monday(hol_date):
+                elif self._is_thursday(hol_date) or self._is_friday(hol_date):
                     hol_date += rd(weekday=MO)
                     hol_name = self.tr("%s (Observado)") % hol_name
             self[hol_date] = hol_name
 
         super()._populate(year)
 
-        # New Year's Day
+        ########################
+        #
+        # FERIADOS INAMOVIBLES
+        #
+        ########################
```

Comment:
```suggestion
        # Fixed date holidays.
```

Reply:
I prefer to have big headers for them in the code instead of a single-liner for different date types header since it's easier for maintenance at a glance - although it doesn't matter much for this case where we only have like, 2 date types:

This would be ideal for me though:
```python
##################
# 
# Fixed Holidays
# 
##################
```

Reply:
I understand your point and totally respect your opinion. Yes, everybody has their own preferences, different coding styles, etc. My goal as a maintainer here is to have 1) a stable (backward compatible) 2) performant library with 3) as readable code base as possible. 

I don't mind accepting the PR as is now. Nevertheless I know that this comment formatting will be changed in a different PR later. I don't think the commenting style needs to be enforced by pre-commit hooks or similar.

I'd like to share some links here to explain my perspective:
  - https://peps.python.org/pep-0008/#comments
  - https://google.github.io/styleguide/pyguide.html#385-block-and-inline-comments

Reply:
It's a valid concern in this case - we may as well fix it now rather than leaving this to future commits.

I'll get them fixed shortly then.